experimental/dds/tree2/src/test/feature-libraries/default-field-kinds/defaultChangeFamily.spec.ts

```diff
@@ -359,6 +365,41 @@ describe("DefaultEditBuilder", () => {
 			expectForest(forest, expected);
 		});
 
+		describe("encodes insert ops using schema based encoding", () => {
+			// testFields were omitted, as we currently do not encode ops with fieldCursors.
```

Comment:
Those cases should not be different in a way where they involve field cursors instead of node cursors. They simple are the only cases that don't have exactly one node at the root, and thus use a different helper to build the tree that supports that.

I think it would make sense to update this test to correctly handle inserts of other numbers of nodes, especially given that cases with more than one node is something who's format we want to optimize, so having test coverage for it would be nice.

Reply:
Fixed test to include different number of nodes, and removed skipped testTrees